adhocracy4/categories/form_fields.py

```diff
@@ -1,34 +1,10 @@
-from collections import abc
-
 from django import forms
+from django.conf import settings
 from django.forms import widgets
-from django.utils.functional import cached_property
 
 from adhocracy4.categories import get_category_icon_url
 
 
-class CategoryIconDict(abc.Mapping):
```

Comment:
I'm not really sure why this was needed so I removed it, I think we should point out to testers to verify everything still works as intended with the icons